docs/admin/snapshots.rst

```diff
@@ -112,6 +112,11 @@ Restore
    If you are restoring a snapshot into a newer version of CrateDB, be sure to
    check the :ref:`release_notes` for upgrade instructions.
 
+.. NOTE::
+
+   Tables that are to be restored must not exist yet while existing metadata
```

Comment:
@seut can you clarify this note? I am not confident that I understand

Reply:
Hm yes, think we should re-phrase.

A table must not exists when someone wants to restore it. The restore operation will fail with an error if the table exist.
Restoring metadata or settings (which is possible with this PR) which already exists will just be overwritten, no error is raised.

Reply:
ah that clears things up thx